packages/ui/core-components/src/lib/atoms/inputs/slider/_Slider.svelte

```diff
@@ -163,6 +163,21 @@
 			}
 		}
 	}
+
+	// handle steps, slider lags when there are greater then 1000 steps/ticks between max and min
+	let sliderTicks = max - min;
+	let minStep = sliderTicks / 1000;
+	console.log(`minStep: ${minStep}`, name);
+	const handleSteps = () => {
+		if (sliderTicks > 1000 && step < minStep) {
+			console.error(`Steps must be at least ${minStep} for slider ${name}`);
+			step = minStep;
+		}
+	};
+
+	$: if (sliderTicks > 1000) {
+		handleSteps();
+	}
```

Comment:
Have you tested this when updating `min`/`max`/`step` reactively? I don't think it will work. Since `sliderTicks` isn't declared with `$:` on line 168, it will never update and so this reactive block will never run.